pkg/kvstore/etcd.go

```diff
@@ -507,7 +509,7 @@ func (e *EtcdClient) GetWatcher(key string, timeSleep time.Duration) <-chan []po
 		for {
 			w := <-e.cli.Watch(ctx.Background(), key, client.WithRev(lastRevision))
 			if w.Err() != nil {
-				log.Warning("Unable to watch key %s, retrying...", key)
+				log.WithField(fieldKey, key).Warn("Unable to watch key, retrying...")
```

Comment:
Remove the ellipsis.

Reply:
Should it use a semicolon, like the other place you suggested it? (so `[...] key; retrying"`)

Reply:
There are also ellipses on the end of a bunch of messages, some of which are Info. Would you like me to remove those too? or was this case specific because it's taking an action?

Reply:
We have been using `...` to indicate that an action is about to start, has not been completed and will be ongoing. @rlenglet What is the motivation to remove this?

Reply:
I just think I’ve seen other places where ellipses were not used in this case. Just wanted to be consistent. No big deal.